packages/pds-ember/addon/components/pds/app/index.hbs

```diff
@@ -7,7 +7,7 @@
   Skip to Content
 </a>
 
-<a id="top" data-test-app-top-anchor></a>
+<a href="#" id="top" data-test-app-top-anchor></a>
```

Comment:
I think we want to put in an ignore for this one - the href-less is better than `href="#"` in this case imo.

Reply:
+1

Reply:
This should be resolved now.